### PR TITLE
[WIP] Fix BioModels tools and include macro

### PR DESCRIPTION
**Pull request by anuprulez**
This PR:
- Make tool ids consistent
- Use Docker container from quay.io/galaxy repository
- Move common things to `macro.xml`

### CODE REVIEW

File: tools/biomodelsML/macros.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<macros>
+    <token name="@PROFILE@">22.05</token>
+    <token name="@VERSION@">1</token>
```
Comment (bgruening) [inline tools/biomodelsML/macros.xml:3]: maybe version 1.1?

**Review by bgruening — COMMENTED** [review]
lgtm, but test are failing

**Comment by anuprulez** [comment]
https://quay.io/repository/galaxy/biomodels_biomd0000001076 The repo was private, and I made it public. The error log says CI did not have permission to pull the container. The functioning container is public: https://quay.io/repository/galaxy/biomodels_biomd0000001066. Let's see if the failing test passes this time.

File: tools/biomodelsML/macros.xml (hunk 1 above)
Comment (anuprulez) [reply tools/biomodelsML/macros.xml]: Ok, added.

**Comment by anuprulez** [comment]
Yes, that was the error related to the permissions. Now, all tests passed. Thanks for reviewing :)

**Pull request merged**